client/js/views/Rules.vue

```diff
@@ -361,15 +373,16 @@ export default {
           {
             title: 'Ace',
             icon: 'delete',
-            description: 'Scrap all point cards on the field',
+            description: 'Scrap all point cards on the field.',
             staticImg: 'cuttle_one_off_ace.png',
             animatedImg:
               'https://github.com/cuttle-cards/cuttle-assets/blob/main/assets/ace.gif?raw=true',
           },
           {
             title: 'Two - Effect 1',
             icon: 'delete',
-            description: 'Twos have two alternative one-off effects: Counter target One-Off Effect',
+            description:
+              "Two's have two alternative one-off effects: Counter target One-Off Effect.  This happens during your opponent's turn.",
```

Comment:
Got an extra space. Not all counters happen on your opponent's turn b/c if you counter their counter then it's still on your turn. Maybe a 'may' effect?
```suggestion
              "Two's have two alternative one-off effects: Counter target One-Off Effect. May be played during your opponent's turn.",
```